src/main/java/com/example/tradingapp/websocket/MyWebsocketClient.java

```diff
@@ -0,0 +1,29 @@
+package com.example.tradingapp.websocket;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+import org.springframework.web.socket.WebSocketHttpHeaders;
+import org.springframework.web.socket.client.WebSocketClient;
+import org.springframework.web.socket.client.standard.StandardWebSocketClient;
+
+import java.net.URI;
+
+@Slf4j
+@Component
+@RequiredArgsConstructor
+public class MyWebsocketClient {
+
+    private final MyWebsocketHandler myWebsocketHandler;
+    private static String URL = "wss://wspap.okx.com:8443/ws/v5/private?brokerId=9999";
```

Comment:
this URL would be different for demo/prod environments and can be moved to properties